docs/releases/1.8.1.txt

```diff
@@ -4,7 +4,8 @@ Django 1.8.1 release notes
 
 *Under development*
 
-Django 1.8.1 fixes several bugs in 1.8.
+Django 1.8.1 fixes several bugs in 1.8 and includes some optimizations in the
```

Comment:
I would like to be more specific about what performance benefits users can expect to see and also include the ticket reference.